Hacktoberfest: Migrate docs from plugins-wiki-docs

avinashupadhya99 opened the pull request:
Migrated docs from [plugins-wiki-docs repo](https://github.com/jenkins-infra/plugins-wiki-docs/tree/master/seleniumhtmlreport) to this repo as a part of Hacktoberfest
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

<!--
Put an `x` into the [ ] to show you have filled the information.
The template comes from https://github.com/jenkinsci/.github/blob/master/.github/pull_request_template.md 
You can override it by creating .github/pull_request_template.md  in your own repository 
-->

avinashupadhya99 committed:
docs: migrate docs from plugins-wiki-docs

MarkEWaite reviewed (changes requested):
Thanks very much for the pull request.

The content looks good.

The file name needs to be changed from `README` to `README.md` so that the file will be rendered as formatted HTML by GitHub and by the Jenkins plugins site.

The `pom.xml` file needs to be changed to refer to this location rather than referring to the wiki.  See [line 14](https://github.com/jenkinsci/seleniumhtmlreport-plugin/blob/b594f45ef282c904df5c17818f70f42b17fe2322/pom.xml#L13)  where the URL needs to be https://github.com/jenkinsci/seleniumhtmlreport-plugin/

See the www.jenkins.io documentation for [alternative values allowed in the pom](https://www.jenkins.io/doc/developer/publishing/documentation/#documenting-plugins).

MarkEWaite commented on README:
```diff
@@ -1,2 +1,34 @@
+support from [Abas Software AG](http://www.abas.de/). Please feel free
+to contact me for feedback. If you have suggestions for new features or
+you've found a bug, please use the [issue
+tracker](http://issues.jenkins-ci.org/browse/JENKINS/component/15781).
```
That link did not work for me.  I've suggested a link that I believe works for users whether they are logged into Jira or operating as an anonymous user.
```suggestion
tracker](https://issues.jenkins.io/issues/?jql=project%20%3D%20JENKINS%20AND%20component%20%3D%20seleniumhtmlreport-plugin).
```

avinashupadhya99 committed:
Update link for issue tracker

Co-authored-by: Mark Waite <mark.earl.waite@gmail.com>

avinashupadhya99 committed:
chore: add extension to README

avinashupadhya99 committed:
chore: update plugin url in pom.xml

avinashupadhya99 committed:
Merge branch 'docs-migration' of github.com:avinashupadhya99/seleniumhtmlreport-plugin into docs-migration

avinashupadhya99 commented:
> The file name needs to be changed from `README` to `README.md` so that the file will be rendered as formatted HTML by GitHub and by the Jenkins plugins site.

I am so sorry, I did not realise the extension was missing, fixed it.

> The `pom.xml` file needs to be changed to refer to this location rather than referring to the wiki. See [line 14](https://github.com/jenkinsci/seleniumhtmlreport-plugin/blob/b594f45ef282c904df5c17818f70f42b17fe2322/pom.xml#L13) where the URL needs to be https://github.com/jenkinsci/seleniumhtmlreport-plugin/

Fixed this as well

MarkEWaite commented on README.md:
```diff
@@ -0,0 +1,34 @@
+support from [Abas Software AG](http://www.abas.de/). Please feel free
+to contact me for feedback. If you have suggestions for new features or
+you've found a bug, please use the [issue
+tracker](https://issues.jenkins.io/issues/?jql=project%20%3D%20JENKINS%20AND%20component%20%3D%20seleniumhtmlreport-plugin).
```
I'm also learning from this experience.  I've found an even better destination link to report an issue.  Thanks for being part of the journey of discovery!

This link opens a page that guides the submitter to correctly classify the issue as a bug, an enhancement request, or a security issue.  If it is a security issue, it also correctly guides them to submit it through the security project so that it is not publicly visible.

This is the same link as the "Report an issue" link on the plugins.jenkins.io page.  Thanks again for your contribution!
```suggestion
tracker](https://www.jenkins.io/participate/report-issue/redirect/#15781).
```
